build-packages/scandipwa-scripts/package.json

```diff
@@ -1,13 +1,14 @@
 {
     "name": "@scandipwa/scandipwa-scripts",
     "description": "Scripts and configuration used by CSA.",
-    "version": "2.2.0",
+    "version": "2.2.1",
```

Comment:
You should NOT bump this version manually!

Reply:
Reverted